lib/components/Note/Note.tsx

```diff
@@ -20,38 +22,56 @@ export const Note = ({
   onCloseClick,
   readMoreText,
   readMoreHref,
-}: NoteProps) => (
-  <div
-    role="alert"
-    aria-live="assertive"
-    aria-atomic="true"
-    className={`
+}: NoteProps) => {
+  const { sm } = useMediaQueries();
+  const hasReadMore = variant === 'success' && readMoreText && readMoreHref;
+  return (
+    <div
+      role="alert"
+      aria-live="assertive"
+      aria-atomic="true"
+      className={`
           ${variant === 'success' ? 'bg-success text-white' : ''}
           ${variant === 'warning' ? 'bg-warning text-black' : ''}
           ${variant === 'error' ? 'bg-alert text-white' : ''}
           `}
-  >
-    <div className="mx-auto flex min-h-11 max-w-[1092px] items-center justify-between pl-7 pr-0">
-      <div className="flex flex-wrap items-end gap-x-5 gap-y-2 py-5 ">
-        <div className="text-heading-4">{title}</div>
-        <div className="text-body-sm">{description}</div>
-      </div>
-      <div className="flex items-center">
-        {variant === 'success' && readMoreText && readMoreHref && (
-          <a
-            className="mx-7 text-nowrap rounded-[30px] bg-white px-6 py-[10px] text-button-md text-success"
-            href={readMoreHref}
-            role="button"
-          >
-            {readMoreText}
-          </a>
-        )}
-        {onCloseClick && (
-          <button className="mr-5 flex" type="button" aria-label="Close" onClick={onCloseClick}>
-            <span className="material-symbols-outlined size-24 select-none p-2">cancel</span>
-          </button>
-        )}
+    >
+      <div className="mx-auto flex min-h-[42px] max-w-[1092px] items-center justify-between px-5 sm:min-h-11 sm:pl-7 sm:pr-0">
+        <div className="flex flex-col flex-wrap items-start py-4 sm:flex-row sm:items-end sm:gap-x-5 sm:gap-y-2 sm:py-5">
+          <div className="text-heading-5 sm:text-heading-4">{title}</div>
+          <div className="mt-1 text-[12px] font-medium leading-[110%] sm:mt-0 sm:text-body-sm">{description}</div>
+          {hasReadMore && !sm && (
+            <a
+              className="mt-4 text-nowrap rounded-[30px] bg-white px-6 py-[10px] text-button-md text-success"
+              href={readMoreHref}
+              role="button"
+            >
+              {readMoreText}
```

Comment:
Maybe not in this ticket, but maybe the **readMoreText** "button" styles should be like normal button to be consistent? (hover, press, etc)

Reply:
Agree. Made [OPHJOD-383](https://jira.eduuni.fi/browse/OPHJOD-383) for it.